script/setup

```diff
@@ -1,53 +0,0 @@
-#!/usr/bin/env sh
-
-# Set up Rails app. Run this script immediately after cloning the codebase.
-#
-# First, you need to create the database user the app will use by manually
-# typing the following in your terminal:
-#
-#   $ sudo -u postgres psql -c "CREATE USER ofn WITH SUPERUSER CREATEDB PASSWORD 'f00d'"
-#
-# This will create the "ofn" user as superuser and allowing it to create
-# databases.
-
-# Exit if any command fails
-set -e
-
-YELLOW='\033[0;33m'
-RED='\033[0;31m'
-NO_COLOR='\033[0m'
-
-RUBY_VERSION=$(cat .ruby-version)
-
-# Check ruby version
-if ! ruby --version | grep $RUBY_VERSION > /dev/null; then
-  printf "${RED}Open Food Network requires ruby ${RUBY_VERSION}${NO_COLOR}. Have a look at: https://github.com/rbenv/rbenv\n"
-  exit 1
-fi
-
-# Set up Ruby dependencies via Bundler
-if ! command -v bundle > /dev/null; then
-  ./script/install-bundler
```

Comment:
This script actually checks the Bundler version, and installs new versions if needed, whereas bin/setup doesn't. I think it would be good to incorporate that to be more idempotent.

Reply:
Somehow I thought that newer version of bundler automatically use the right version but that doesn't seem to be true. I misremembered the logic of https://bundler.io/guides/bundler_2_upgrade.html#version-autoswitch.

So yes, we should run this as well.